design-proposals/eim-decouple-microos-storage.md

```diff
@@ -0,0 +1,185 @@
+# Design Proposal: Decouple MicroOS storage from the orchestrator
+
+Author(s): Tomasz Osiński
+
+Last updated: 24.04.2025
+
+## Abstract
```

Comment:
another benefit to list is also the startup time of DKAM - given all the issues we faced with the readiness probe implementation